### DISCUSSION TITLE
Unable to view the feedback of the mentor on my final project submission

### DISCUSSION THREAD

**Discussion by adtygan** [OpenMined/courses#228] [Course: Our Privacy Opportunity] [answered]
So I got a mail stating a Failed Review of my Final Project submission. But I'm unable to see the feedback the mentor gave me. If I can see that, I can make the appropriate changes. I'd be happy if someone could help me out with this.

**Comment by yemikifouly** [comment, marked as answer]
Hello Aditya! 👋 This is very odd. I am looking into this. I will get back to you in a few minutes.

**Reply by adtygan** [reply]
Sure, thanks!

**Reply by yemikifouly** [reply]
Aditya, I tracked the problem. It seems that the reviewer had an issue on their end and the review came back to you with no feedback. Could you please, copy and paste your original submission in a google doc and share it with me on Slack? My handle is @Yemi. I will make sure you are sent the feedback that the reviewer meant to share so that you can fix your submission accordingly. 🙂 I am not suggesting you resubmit through the classroom because I don't want you to lose one attempt because of us. 🙂
